README.md

```diff
@@ -32,6 +32,7 @@
 
 - **`sprocket analyzer`** runs Sprocket as a LSP server, which is useful for IDE integration.
 - **`sprocket check`** performs static analysis on a document or directory of documents.
+- **`sprocket config`** prints effective configuration.
```

Comment:
This doesn't account for the `init` option. Maybe this should be more general to future proof? Something like "prints configuration values"?